### ISSUE TITLE
[Accessibility]: Commits page - User profile images alt text

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34974]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34423

### Problem description

<img width="110" alt="image" src="https://user-images.githubusercontent.com/9516420/166909519-07347a08-b182-4237-8259-2d99c93ce6c8.png">

Default user image: "Reads KR" - Not useful to the user
Provided user image - No alt text.

### Expected behavior

In this area it would be OK to make _all_ of these images `aria-hidden`.

However, there is no harm by adding alt text here - and I believe these images are used in many places so it might be the safest option overall.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Will be closed by https://github.com/sourcegraph/sourcegraph/pull/35617
